### DISCUSSION TITLE
RGB Coloring of Point Clouds

### DISCUSSION THREAD

**Discussion by banesullivan** [pyvista/pyvista#6437] [Q&A]
## Description

From @tomtomatron:

>Anyone have tips for efficiently plotting a large colored point cloud (each point having it's own RGB value in a tuple or array) using pyvista?  

## Example Data

> Trying to visualize something like this but with points and not a mesh: https://sketchfab.com/3d-models/briery-gap-k1-n-0c81186d72f54a398e85c1f492da625a

**Comment by banesullivan** [comment]
Download the data from that link then:

```py
import pyvista as pv
from pyvista import examples
import numpy as np
import os

path = '/Users/bane/Downloads/briery-gap-k1-n/source/model'
mesh = pv.read(os.path.join(path, 'model.obj'))
mesh.plot(notebook=False)
```

<img width="624" alt="Screen Shot 2019-06-28 at 1 45 14 PM" src="https://user-images.githubusercontent.com/22067021/60367259-068ef400-99ab-11e9-8443-a972e8662a43.png">


And then add your RGB array of shape (`n_points` by 3):

```py
mesh['colors'] = my_rgb_array
mesh.plot(scalars='colors', rgb=True, notebook=False)
```

**Comment by banesullivan** [comment]
There is a texture included with that example, but what is the mapping?

**Comment by tzodge** [comment]
If I open obj files with MeshLab, I can see the colors.

Can not see them, when I load them with pyvsita :/.
What am I missing ?
![Screenshot from 2021-06-16 15-35-25](https://user-images.githubusercontent.com/37312027/122281651-7eb49400-ceb8-11eb-8dc1-7ebbf603e3b0.png)


![Screenshot from 2021-06-16 15-33-45](https://user-images.githubusercontent.com/37312027/122281578-6775a680-ceb8-11eb-9530-e7d7fabd2765.png)

Here's a link to the entire folder with material and everything
[002_master_chef_can.zip](https://github.com/pyvista/pyvista-support/files/6665407/002_master_chef_can.zip)

**Comment by ArghyaChatterjee** [comment]
Is there any update to this issue ?? I am having the same issue. I can't visualize the RGB colored pointcloud, my pointcloud is of .ply type.
